cross-platform/react-app/package.json

```diff
@@ -12,8 +12,9 @@
     "start": "cross-env TRANSPILE_DEPS=false DISABLE_TERSER=true USE_FAST_SASS=true BROWSER=none react-scripts start",
     "start:camera": "cross-env REACT_APP_BUILD_TARGET=Camera TRANSPILE_DEPS=false DISABLE_TERSER=true USE_FAST_SASS=true BROWSER=none react-scripts start",
     "copy:assets": "cpx \"./node_modules/@itwin/mobile-sdk-core/lib/public/locales/**\" ./build/locales && cpx \"./node_modules/@itwin/mobile-ui-react/lib/public/locales/**\" ./build/locales",
+    "copy:camera": "cpx \"./src/frontend/CameraSample/public/locales/**\" ./build/locales",
```

Comment:
I wonder if ./src/frontend/CameraSample/public should be added to the exclude array in tsconfig.json. It's obviously not necessary, but I wonder if it would be good to do anyway.

Reply:
I have no idea. Since it contains json files, they won't be processed by tsc anyway, will they?

Reply:
Presumably not.